package.json

```diff
@@ -1,9 +1,17 @@
 {
   "private": true,
   "scripts": {
-    "format": "prettier --single-quote --trailing-comma=es5 --print-width=100 --write '{create-react-native-app,react-native-scripts}/{src,template}/**/*.js'"
+    "format": "prettier --single-quote --trailing-comma=es5 --print-width=100 --write '{create-react-native-app,react-native-scripts}/{src,template}/**/*.js'",
+    "postinstall": "opencollective postinstall"
```

Comment:
The postinstall script would have to go in `./create-react-native-app/package.json` (not the internal package.json at the root) or it won't be triggered when people install the package.

Also the message that's shown when I run Yarn looks a bit messy / badly aligned :
<img width="445" alt="screen shot 2018-02-24 at 12 06 32" src="https://user-images.githubusercontent.com/497214/36629539-a800aff4-195f-11e8-8fb9-66cdd6f91d95.png">

If the postinstall script is optional, maybe we could just leave it out for now?

Reply:
The post install is not ideal for all repos - especially if you think that your repo will be bundled in another, users will see it every time. You can ignore or roll your own. 
One user made a lighter version of the post install: https://github.com/WebReflection/lightercollective
You can also remove the dependency altogether: https://github.com/remy/nodemon/blob/master/package.json#L42